### ISSUE TITLE
Post meeting spatial model exploration

### ISSUE THREAD

**Issue by brianlangseth-NOAA** [brianlangseth-NOAA/Spatial-Workshop-SPASAM#57]
From @JDeroba:

Quick update. 1) Estimating init_abund took 3 hours and the likelihood actually got worse, suggesting a local minima. The fit to the index and gradient improved, but Fleet 2 selectivity went wonky. I think I can fix all this with different starting values or phases. But for now I'm going to pursue other explorations and then revisit this when we've checked some other boxes (e.g. explored movement). 2) I estimated recruitment apportionment. The fit took 2 hours and had an amazing gradient. I think correctly, the model puts 99.3% of recruits in area1 and 0.7% in area2. The fit to the early years of the index was much improved, but otherwise the diagnostics were similar. The movement parameters were similar between this run and the run without approtionment. So I see no evidence of confounding. I think I'm going to fix this apportionment at 99% and 1%. I recall another group doing something similar. I suspect it'll save time and what sense is there in estimating 99%? Again, once I try some movement stuff, I can try this again in a final attempt to estimate a "super model". I'm happy to hear your input.

**Comment by brianlangseth-NOAA** [comment]
@brianlangseth-NOAA reply:
Your choices sound very reasonable. Estiamting init_abund early on with the single area model only resulted in weird resutls. I recall you added it near the end of your explorations and only then did it behave well. Conseuqently I think your choice to wait is wise. 

I did not think to estimate recruitment apportionment. Nice idea. Another group fixed that at 100% to area 1, so this aligns with someone elses choice. Was recrutiment apportionment yearly, or constant across years? Fitting better early on could make sense in that earlier the population was larger, and thus higher reruitment (i.e. less likely to be on the descending limb of the bev-holt). That is why it could have more of an early effect.

I think the choice to fix apportionment at 99% is perfectly reasonable, with the justriciation that early exploration suggested this was the value.

**Comment by JDeroba** [comment]
@brianlangseth-NOAA Using the 2area, 4fleet setup, I fixed rec apportionment as noted in this thread previously (99% area 1). I then estimated movement for two age groups (immature <=9 and mature >9) in 2 year blocks of pseudo year. So # pseudo-year/2 * 2 movement parameters (This run is YFT_2area_4fleets_F0.0001_18alk). This fit in less than 2 hours with good gradient and great fit. I then I tried to estimate init_abund again, but had the same problems as noted previously in this thread. I then tried to estimate movement for the two age groups for every psuedo-year. This produced similar fits to estimating movement in two year blocks (...18alk), and so offered no advantage. Best run is YFT_2area_4fleets_F0.0001_18alk and I suggest we try it with the 100 datasets. Lastly, I tried a 4 area, 7 fleet model with the tagging data ESS =20 as in the 2area/4fleet. i also tried to set rec apportionment to 96% area1, but that didn't work for some reason. In any case, this fit took ~30 hours, OK gradient, wonky selectivity, still trends in survey residuals in most areas, and generally similar scales and trends as the 2 area/4fleet model. So I see no reason to keep pursuing this higher dimension. I'll upload everything in a commit shortly including a "ChangeLog.txt" that describes each model variant and the process.
